MusicApp.SongService.Application/CQRS/Queries/GetSongs/GetSongsQueryHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using MediatR;
+using MusicApp.SongService.Application.Repositories;
+using MusicApp.SongService.Domain.Entities;
+
+namespace MusicApp.SongService.Application.CQRS.Queries.GetSongs;
+
+public class GetSongsQueryHandler : IRequestHandler<GetSongsQuery, IEnumerable<Song>>
+{
+    private readonly ISongRepository _repository;
+
+    public GetSongsQueryHandler(ISongRepository repository)
+    {
+        _repository = repository;
+    }
+
+    public async Task<IEnumerable<Song>> Handle(GetSongsQuery request, CancellationToken cancellationToken)
+    {
+        return await _repository.GetSongsAsync(cancellationToken);
```

Comment:
maybe you should return list of DTOs  here?

Reply:
I thought about it, but I can't think of what DTO I should do. After all, all the fields of this object will be needed on the client.

Reply:
okay, leave it as it is